tokio/tests/time_sleep.rs

```diff
@@ -267,6 +267,19 @@ async fn exactly_max() {
     time::sleep(ms(MAX_DURATION)).await;
 }
 
+#[tokio::test]
+async fn issue_5183() {
+    time::pause();
+
+    let big = std::time::Duration::from_secs(u64::MAX / 10);
+    // This is a workaround since awaiting sleep(big) will never finish.
+    tokio::select! {
+    biased;
```

Comment:
```suggestion
        biased;
```

Reply:
I had it like that at first, but rustfmt disagree with it, so I have skipped the expression now.

Reply:
Huh? Rustfmt usually does not format macros. Can you try and remove the skip annotation to see if our CI fails it?

Reply:
I did hence the force push above, here is the specific run of the CI
https://github.com/tokio-rs/tokio/actions/runs/5229606790/jobs/9442690852

Reply:
Weird. Oh well.